Fails to build with GDAL 2.5.0

The liblas Debian package failed to build with GDAL 2.5.0-beta1:
```
/build/liblas-1.8.1/src/gt_wkt_srs.cpp:496:18: error: 'class OGRSpatialReference' has no member named 'FixupOrdering'
             oSRS.FixupOrdering();
                  ^~~~~~~~~~~~~
```

From https://github.com/OSGeo/gdal/blob/v2.5.0beta1/gdal/MIGRATION_GUIDE.TXT:
>    * removal of OSRFixup() and OSRFixupOrdering(): no longer needed since objects
>      constructed are always valid

After removing `oSRS.FixupOrdering ` from 1.8, I am greeted with this error:

```
root@f4381c014d4e:~/libLAS-1.8.1/makefiles# make
[  1%] Linking CXX shared library ../bin/Release/liblas.so
[ 41%] Built target las
[ 42%] Linking CXX shared library ../bin/Release/liblas_c.so
[ 44%] Built target las_c
[ 45%] Linking CXX executable ../bin/Release/lasinfo
/usr/bin/ld: CMakeFiles/lasinfo.dir/laskernel.cpp.o: undefined reference to symbol '_ZNSt15basic_streambufIcSt11char_traitsIcEE8overflowEi@@GLIBCXX_3.4'
/usr/bin/ld: /usr/lib/gcc/x86_64-linux-gnu/9/libstdc++.so: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[2]: *** [apps/CMakeFiles/lasinfo.dir/build.make:112: bin/Release/lasinfo] Error 1
make[1]: *** [CMakeFiles/Makefile2:281: apps/CMakeFiles/lasinfo.dir/all] Error 2
make: *** [Makefile:163: all] Error 2
```

Any idea why?

--- a/src/gt_citation.cpp
+++ b/src/gt_citation.cpp
@@ -34,7 +34,9 @@
 #include "geovalues.h"
 #include "gt_citation.h"
 
+#if GDAL_VERSION_NUM < GDAL_COMPUTE_VERSION(3,12,0)
 CPL_CVSID("$Id: gt_citation.cpp 21848 2011-02-25 16:41:38Z warmerdam $")
+#endif
 
 static const char *apszUnitMap[] = {
     "meters", "1.0",

--- a/src/gt_wkt_srs.cpp
+++ b/src/gt_wkt_srs.cpp
@@ -45,7 +45,9 @@
 #include "gt_wkt_srs_for_gdal.h"
 #include "gt_citation.h"
 
+#if GDAL_VERSION_NUM < GDAL_COMPUTE_VERSION(3,12,0)
 CPL_CVSID("$Id$")
+#endif
 
 #define ProjLinearUnitsInterpCorrectGeoKey   3059
 

--- a/src/gt_wkt_srs.cpp
+++ b/src/gt_wkt_srs.cpp
@@ -301,7 +301,9 @@ char *GTIFGetOGISDefn( GTIF *hGTIF, GTIFDefn * psDefn )
                 oSRS.SetFromUserInput(pszWKT);
                 oSRS.SetExtension( "PROJCS", "PROJ4",
                                    "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0 +k=1.0 +units=m +nadgrids=@null +wktext  +no_defs" );
+#if GDAL_VERSION_NUM < GDAL_COMPUTE_VERSION(2,5,0)
                 oSRS.FixupOrdering();
+#endif
                 CPLFree(pszWKT);
                 pszWKT = NULL;
                 oSRS.exportToWkt(&pszWKT);
@@ -507,7 +509,9 @@ char *GTIFGetOGISDefn( GTIF *hGTIF, GTIFDefn * psDefn )
         {
             char	*pszWKT;
             oSRS.morphFromESRI();
+#if GDAL_VERSION_NUM < GDAL_COMPUTE_VERSION(2,5,0)
             oSRS.FixupOrdering();
+#endif
             if( oSRS.exportToWkt( &pszWKT ) == OGRERR_NONE )
                 return pszWKT;
         }
@@ -1109,7 +1113,9 @@ char *GTIFGetOGISDefn( GTIF *hGTIF, GTIFDefn * psDefn )
 /* ==================================================================== */
     char	*pszWKT;
 
+#if GDAL_VERSION_NUM < GDAL_COMPUTE_VERSION(2,5,0)
     oSRS.FixupOrdering();
+#endif
 
     if( oSRS.exportToWkt( &pszWKT ) == OGRERR_NONE )
         return pszWKT;

